components/purchase_modal/process_payment_setup.tsx

```diff
@@ -139,6 +143,45 @@ export default class ProcessPaymentSetup extends React.PureComponent<Props, Stat
         this.props.onBack();
     }
 
+    private sucessPage = () => {
+        const {error} = this.state;
+        if (this.props.isProratedPayment) {
+            const formattedButonText = (
+                <FormattedMessage
+                    defaultMessage={'Return to {team}'}
+                    id={'admin.billing.subscription.proratedPayment.buttonText'}
+                    values={{team: this.props.teamName}}
+                />
+            );
+            return (
+                <>
+                    <IconMessage
+                        title={t('admin.billing.subscription.proratedPayment.title')}
+                        subtitle={t('admin.billing.subscription.proratedPayment.substitle')}
+                        date={getNextBillingDate()}
+                        error={error}
+                        icon={successSvg}
+                        formattedButonText={formattedButonText}
+                        buttonHandler={this.props.onClose}
+                        className={'success'}
+                    />
+                </>
```

Comment:
In this success page we are still missing the table of the example of prorated?

<img width="588" alt="Screen Shot 2021-07-15 at 10 25 13 PM" src="https://user-images.githubusercontent.com/936315/125882181-484a5455-3492-43be-b63f-3345c833021e.png">

Reply:
This will be done as part of the ticket https://mattermost.atlassian.net/browse/MM-37164

Reply:
Sounds good!